ros/lib/models.py

```diff
@@ -55,6 +55,7 @@ class System(db.Model):
     rule_hit_details = db.Column(JSONB)
     state = db.Column(db.String(25))
     number_of_recommendations = db.Column(db.Integer)
+    stale_timestamp = db.Column(db.DateTime)
```

Comment:
If you're setting `timezone=True` in the migration, then it should come from the model.
```suggestion
    stale_timestamp = db.Column(db.DateTime(timezone=True))
```